public/locales/af/common.json

```
@@ -0,0 +1,17 @@
+{
+    "submit": "Indien",
```

```suggestion
    "submit": "Dien in",
```
I think its the other way around